### DISCUSSION TITLE
Reset password: The requested resource wasn't found.

### DISCUSSION THREAD

**Discussion by lekoshimura** [pocketbase/pocketbase#3369] [Q&A] [answered]
PocketBase v0.18.6

* localhost
* fresh install
* orignal email templates

Steps to reproduce:

* I configured "Mail Settings"
* Sent a password request test email
* I got the message in my inbox and clicked "Reset password"
* I filled the new password form and submited it.
* "The requested resource wasn't found" error is shown.

The form is posting to:

`http://localhost:8090/api/collections/__pb_test_collection_id__/confirm-password-reset`

resulting in 404 status code.

**Comment by ganigeorgiev** [comment]
We have both api (see [`record_auth_test.go`](https://github.com/pocketbase/pocketbase/blob/f605521208179072a1251042e2b16835fcc955cc/apis/record_auth_test.go#L491)) and form (see [`record_password_reset_confirm_test.go`](https://github.com/pocketbase/pocketbase/blob/f605521208179072a1251042e2b16835fcc955cc/forms/record_password_reset_confirm_test.go)) automated tests that confirm the correct behavior.

Do you use a custom confirmation url? How are you sending the confirm request?

**Reply by ganigeorgiev** [reply, marked as answer]
Oh, you are sending test email. In this case is expected to fail because the test email is not associated to any real user, aka. the email and the token is "dummy" just for test purposes.

**Reply by lekoshimura** [reply]
Oh! Silly me! I'm sorry for bothering you with it. And thank you very much for your amazing work, @ganigeorgiev !
